src/Command/AbstractCommand.php

```diff
@@ -61,7 +61,7 @@ protected function getPackageTitle($install_version = null)
     {
         $package = self::PACKAGE_NAME;
         if (!is_null($version = $install_version)) {
-            $package .= ":$version";
+            $package .= ":^$version";
```

Comment:
This change is already on master.
https://github.com/pantheon-systems/terminus-installer/blob/master/src/Command/AbstractCommand.php#L64

Reply:
I have no idea why | how this happen? maybe it has something to do with rebase I did. I think we can look at this together and walk me through the revert + rebase process